festipals/src/components/Pals.js

```diff
@@ -1,10 +1,13 @@
 import React, { Component } from 'react';
+import samplePals from './../samplePals.json';
 
 export default class Pals extends Component {
 
   render() {
 
+    var pals = samplePals.user.pals;
 
+    pals.foreach(function(pal) {
```

Comment:
This loop is unnecessary, because we're already mapping the list of users on the PalsPage. One 'Pals' component should only be 1 panel, so on the PalsPage we make a list of 'Pals'. This is the same with the 'Acts' on the ActsPage. Sorry if my naming is confusing! Maybe it should be changed, as 'Pal' and 'Act' would make more sense in this case.